README.md

```diff
@@ -62,11 +62,12 @@ state files).
 
 ### Deploy
 
-Run the following script with the name of your environment:
+Create the bootstrap Concourse with `make`. Select the target based on which AWS account you want to work with. For instance for a DEV bootstrap:
 
 ```
-./vagrant/deploy.sh <deploy_env>
+make dev-bootstrap
```

Comment:
There's no mention of setting `DEPLOY_ENV` prior to this, which is required when using the make target. It would be good to cover this detail.